models/ride_models.py

```diff
@@ -20,9 +18,10 @@ def __init__ (self,location,destination):
 
 class DriverOffer:
     """Driver offers ride"""
```

Comment:
Be consistent on comments. Some of your classes have comments while others do not.